### DISCUSSION TITLE
Website Development

### DISCUSSION THREAD

**Discussion by vrm-piyush** [github/india#637] [GitHub Platform]
I have a doubt regarding the website development.
Should we develop a website separately nad deploy it on Microsoft Azure ?
Or should we build it on azure itself?

**Comment by kitarp29** [comment]
How can you develop a website in Azure itself?
Azure is a cloud providing service, not a langauage.

**Comment by FRTMentor** [comment]
Please develop a website with any platform which you're familiar.
And deploy it on the Azure.
Please go through the some YouTube links for more understanding.
